beefy-gadget/src/lib.rs

```diff
@@ -284,15 +284,20 @@ where
 				validator_set_id: 0,
 			};
 
+			// TODO: this is really ugly due to the recent signature change of `sign_with()`.
```

Comment:
Can you add an issue number to this `TODO`? So that it's not forgotten?

Reply:
Done #92